config/betterquesting/DefaultQuests.json

```diff
@@ -306452,6 +306452,415 @@
+          "groupDetect:1": 0,
+          "ignoreNBT:1": 0,
+          "index:3": 1,
+          "consume:1": 0,
+          "requiredItems:9": {
+            "0:10": {
+              "id:8": "TConstruct:swordBlade",
+              "Count:3": 1,
+              "Damage:2": 14,
+              "OreDict:8": ""
+            },
+            "1:10": {
+              "id:8": "TConstruct:toolRod",
+              "Count:3": 1,
+              "Damage:2": 14,
+              "OreDict:8": ""
+            },
+            "2:10": {
+              "id:8": "TConstruct:crossbar",
+              "Count:3": 1,
+              "Damage:2": 14,
+              "OreDict:8": ""
+            }
+          },
+          "taskID:8": "bq_standard:retrieval"
+        }
+      },
+      "rewards:9": {
+        "0:10": {
+          "rewardID:8": "bq_standard:choice",
+          "index:3": 0,
+          "choices:9": {
+            "0:10": {
+              "id:8": "TConstruct:materials",
+              "Count:3": 4,
+              "Damage:2": 14,
+              "OreDict:8": ""
+            },
+            "1:10": {
+              "id:8": "enhancedlootbags:lootbag",
+              "Count:3": 1,
+              "Damage:2": 1,
+              "OreDict:8": ""
+            }
+          }
+        },
+        "1:10": {
+          "rewardID:8": "bq_standard:item",
+          "index:3": 1,
+          "rewards:9": {
+            "0:10": {
+              "id:8": "dreamcraft:item.CoinSmith",
+              "Count:3": 5,
+              "Damage:2": 0,
+              "OreDict:8": ""
+            },
+            "1:10": {
+              "id:8": "dreamcraft:item.CoinSurvivor",
+              "Count:3": 5,
+              "Damage:2": 0,
+              "OreDict:8": ""
+            },
+            "2:10": {
+              "id:8": "dreamcraft:item.CoinTechnician",
+              "Count:3": 5,
+              "Damage:2": 0,
+              "OreDict:8": ""
+            }
+          }
+        }
+      }
+    },
+    "2898:10": {
+      "questID:3": 2898,
+      "preRequisites:11": [
+        486
+      ],
+      "properties:10": {
+        "betterquesting:10": {
+          "snd_complete:8": "random.levelup",
+          "taskLogic:8": "AND",
+          "partySingleReward:1": 0,
+          "visibility:8": "NORMAL",
+          "isMain:1": 0,
+          "simultaneous:1": 0,
+          "icon:10": {
+            "id:8": "TConstruct:jerky",
+            "Count:3": 1,
+            "Damage:2": 5,
+            "OreDict:8": ""
+          },
+          "snd_update:8": "random.levelup",
+          "repeatTime:3": -1,
+          "globalShare:1": 1,
+          "questLogic:8": "AND",
+          "repeat_relative:1": 1,
+          "name:8": "Crispy Rotten Flesh",
+          "isGlobal:1": 0,
+          "lockedProgress:1": 0,
+          "autoClaim:1": 0,
+          "isSilent:1": 0,
+          "desc:8": "Rotten Flesh is not really healthy. It doesn\u0027t satisfy your hunger. What if you try and dry it?"
```

Comment:
Was drying things explained before this? Was there a quest for the drying rack? Otherwise, how do people know how to dry things? Well, unless they've done it in another pack.

Reply:
it pops up after you make tanned leather using a drying rack